cli/repair/repair.go

```diff
@@ -42,6 +42,8 @@ func SetLeader(ctx *context.Ctx) error {
 func Run(processConfFunc ProcessConfFuncType, ctx *context.Ctx, patchConf bool) error {
 	log.Debugf("Data directory is set to: %s", ctx.Running.DataDir)
 
+	// XXX: reload don't work for cartridge < 2.0
```

Comment:
Please provide "Not supported for ..." error message for such case.

Reply:
I'm writing the code that checks it right now =)